packages/components/src/toggle-group-control/toggle-group-control/toggle-group-control-backdrop.tsx

```diff
@@ -1,86 +1,96 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import { useReducedMotion } from 'framer-motion';
+
 /**
  * WordPress dependencies
  */
-import { useState, useEffect, memo } from '@wordpress/element';
+import { useLayoutEffect, useState, memo } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import type { ToggleGroupControlBackdropProps } from '../types';
-import { BackdropView } from './styles';
+import { useToggleGroupControlContext } from '../context';
+import { CONFIG } from '../../utils';
+import { AnimatedBackdrop } from './styles';
+
+const TRANSITION_CONFIG = {
+	type: 'spring',
+	bounce: 0.2,
+	// Transition durations in the config are expressed as a string in milliseconds,
+	// while `framer-motion` needs them as integers in seconds.
+	duration: parseInt( CONFIG.transitionDurationFast, 10 ) / 1000,
```

Comment:
This felt a bit hacky but I couldn't come up with another way of reusing our config as it's written today